overlay/overlay.nix

```diff
@@ -33,6 +33,9 @@ in
     msm-fb-refresher = callPackage ./msm-fb-refresher { };
     msm-fb-handle = callPackage ./msm-fb-handle { };
     ply-image = callPackage ./ply-image { };
+    pulseaudio-modules-droid = callPackage ./pulseaudio-modules-droid { };
+    xf86-video-hwcomposer = callPackage ./xf86-video-hwcomposer { };
```

Comment:
The `xf86` package will need to:

 1. move into the `xorg` "namespace".
 1. be renamed dashless.

This is so it can compose properly with the [nixos module](https://github.com/NixOS/nixpkgs/blob/b3772959864ee1c3fb51c5c14a71ba205c517b9f/nixos/modules/services/x11/xserver.nix#L253-L258).

It should look like this approximately:

```
   xorg = super.xorg.overrideScope'(self: super: {
     xf86videohwcomposer = callPackage ./xf86-video-hwcomposer { };
   }) # See all-packages.nix for more about this messy composition :/
   // { inherit (self) xlibsWrapper; };
```

The last bit is the non-obvious part :). That's what I've been using with the `fbdev` hack patch in other WIP branches.